deploy/crds/pulumi.com_programs.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.15.0
+    controller-gen.kubebuilder.io/version: v0.14.0
```

Comment:
We should find a way to sync up our dev dependencies so we don't oscillate between versions.

Reply:
Still, I agree that the makefile isn't doing enough here and will invesigate.